shared/js/dataDriver.js

```diff
@@ -149,6 +151,12 @@ loop.DataDriver = function() {
             console.log(`Error handling EventSource.put: ${event.data}`, ex);
           }
           break;
+        case "error":
+          setTimeout(
+            this._connectToRoom.bind(this, this._roomToken),
```

Comment:
On a larger question.. do we actually want `_connect` to do all of what it has already done? E.g., refetching all of old chat?